core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -71,6 +71,8 @@ statement
         RENAME COLUMN (IF EXISTS)? from=identifier TO to=identifier    #renameColumn
     | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
         DROP COLUMN (IF EXISTS)? column=qualifiedName                  #dropColumn
+    | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
+        ALTER COLUMN columnName=identifier SET DATA TYPE type          #setColumnType
```

Comment:
`DROP COLUMN` and `RENAME COLUMN` support a column `IF EXISTS`, should we do the same here?

Reply:
i don't know what it is for.

Reply:
Looks like @kokosing did the review for the column `(IF EXISTS)?` https://github.com/trinodb/trino/pull/4651 Maybe he has an opinion. 

Doesn't seem like a blocker though